### PR TITLE
Add release how-to

### PR DESCRIPTION
**Pull request by OndraM**
Ref. AR-129

This is to help others follow proper release process, in case of me being away 🏝 🍹.

### CODE REVIEW

**Review by legendik — APPROVED** [review]

File: RELEASE.MD [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Releasing new Matej PHP Client version
+
+1. Check [open pull-requests](https://github.com/lmc-eu/matej-client-php/pulls), if there are any, you may coordinate releasing of a new version with authors of these pull requests.
```
Comment (MortalFlesh) [inline RELEASE.MD:3]: why are there only `1.` ?

File: RELEASE.MD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - `git commit -m "Release version X.Y.Z"`
+1. Tag the release:
+    - `git tag X.Y.Z`
+1. Push the changes in master branch and tags to the repository:
+    - `git push origin master`
+    - `git push origin X.Y.Z`
+1. Now the [PHP 5 version](https://github.com/lmc-eu/matej-client-php5) of the client needs to be updated and tagged as well:
+    1. Clone the repo: `git clone git@github.com:lmc-eu/matej-client-php5.git` or pull the master branch if you have it already cloned: `git pull origin master`
+    1. Inside the repository directory run transpilation script: `./transpile.sh`
+    1. Make sure the whole process completed successfully, including passed unit tests.
```
Comment (MortalFlesh) [inline RELEASE.MD:24]: `Changelog` of the [PHP 5 version] should be updated as well.

**Review by MortalFlesh — CHANGES_REQUESTED** [review]

File: RELEASE.MD (hunk 1 above)
Comment (jirinovak) [reply RELEASE.MD:3]: It's markup or github feature I guess. When you add some new bullets you have not manually renumber.

File: RELEASE.MD (hunk 2 above)
Comment (jirinovak) [reply RELEASE.MD:24]: It's not true, [PHP5 version] does not contain `Changelog` it instead link the one from this repo.

**Review by jirinovak — APPROVED** [review]

**Review by MortalFlesh — APPROVED** [review]

File: RELEASE.MD (hunk 1 above)
Comment (OndraM) [reply RELEASE.MD:3]: Every time I use this markdown numbering, someone asks this question :-D. It seems like not so know markdown feature 🤔.

Anyway you can check the markdown result here:  https://github.com/lmc-eu/matej-client-php/blob/30ee967130058c17d89bf0fd60cba3809d186285/RELEASE.MD

**Pull request merged**